examples/src/main/kotlin/example/ForwardingRefs.kt

```diff
@@ -0,0 +1,41 @@
+package example
+
+import org.w3c.dom.HTMLInputElement
+import react.*
+import react.dom.html.ReactHTML.input
+
+external interface Focusable {
+    fun focus()
+}
+
+val FancyInput = forwardRef<Props> { _, forwardedRef ->
+    val inputRef = useRef<HTMLInputElement>()
+
+    useImperativeHandle(forwardedRef, inputRef) {
+        object : Focusable {
+            override fun focus() {
+                inputRef.current?.focus()
+            }
+        }
+    }
+
+    input {
+        attrs {
```

Comment:
`attrs` can be removed
`react.dom.html.ref` - Strict ref

Reply:
in `React` ref is regular prop, I don't know why it was moved to `RElementBuilder`